common/compositor/va/varenderer.cpp

```diff
@@ -294,6 +367,67 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
   param.filters = nullptr;
   param.filter_flags = VA_FRAME_PICTURE;
 
+  if(caps[0].type == VAProcColorBalanceNone) {
+      if (!QueryVAProcFilterCaps(
+          va_context, VAProcFilterColorBalance,
+	 caps, &cap_num) || !SetVAProcFilterDefaultValue(&caps[0]) ) {
+              return false;
+      }
+  }
+
+  ScopedVABufferID filter_hue(va_display_);
+  ScopedVABufferID filter_sturation(va_display_);
+  ScopedVABufferID filter_brightness(va_display_);
+  ScopedVABufferID filter_contrast(va_display_);
+  std::vector<VABufferID> filters;
+  VAProcFilterParameterBufferColorBalance cbparam;
+
+ SetVAProcFilterValue(VAProcColorBalanceBrightness, 100);
+
+  cbparam.type = VAProcFilterColorBalance;
+
+  if(hue_value != GetVAProcFilterDefaultValue(VAProcColorBalanceHue, &caps[0]) ) {
```

Comment:
You can't compare two float values.  http://www.geeksforgeeks.org/comparison-float-value-c/. Can we change it to int?